requirements/docs.txt

```diff
@@ -1,13 +1,6 @@
-backports-abc==0.5
-Click==8.1.3
-Jinja2==3.1.2
-livereload==2.6.3
-Markdown==3.3.7
-mkdocs==1.4.3
-mkdocs-bootstrap==1.1
-mkdocs-bootswatch==1.1
-mkdocs-rtd-dropdown==1.0.2
-pymdown-extensions==10.0.1
-PyYAML==6.0
-singledispatch==3.7.0
-tornado==6.1
+#
+# This file is autogenerated by pip-compile with Python 3.8
+# by the following command:
+#
+#    pip-compile --config=pyproject.toml --generate-hashes requirements/docs.in
+#
```

Comment:
Is this file empty because docs.in is an empty file? Is that intentional, or are those requirements missing?

Reply:
No, I seem to have left those requirements out! That's a mistake.

Reply:
Fixed in 621dc5c. I've limited the pins there to what we use, and the `docs.txt` will have the full set of packages required.